lib/constants/strings/asset_constants.dart

```diff
@@ -0,0 +1,15 @@
+class AssetConstants {
+  static const String dalle = 'assets/images/dalle.png';
+  static const String icLink = 'assets/images/ic_link.svg';
+  static const String icForward = 'assets/images/ic_forward.svg';
+  static const String icForward30 = 'assets/images/ic_forward_30.svg';
```

Comment:
I think these icons already exist as Icons.forward_30_sharp etc

Reply:
I saw some minor changes in the icForward30 and icReplay10 that's why I added a new one. I will remove the previously imported icons.